zproject/urls.py

```diff
@@ -745,6 +746,11 @@
     path("api/v1/events/internal", get_events_internal),
 ]
 
+# TusD views
+urls += [
+    rest_path("tusd/hooks", POST=handle_tusd_hook),
+]
```

Comment:
Currently this is open to a normal user, have to come up with a solution. If we use path instead of rest_path, than we have to do calculations on Cookies and other headers to find out users etc.